### ISSUE TITLE
Allow script to operate w/ label

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cberetta/gmail-delay-send#18]
```
A user might like to forgo the step to add a label (eg. 'DELAY_SEND'). And just 
send things directly from their draft folder.

Entering "None" or deleting the label should allow this.
```

Original issue reported on code.google.com by `blairk...@gmail.com` on 27 Nov 2011 at 4:32

**Comment by GoogleCodeExporter** [comment]
```
Title should be "Allow script to operate w/o label"

Fixed in version 0.7.4 (sneak peak: 
https://docs.google.com/spreadsheet/ccc?key=0AgKSjMWgSECadDhtVVlOUXFLTmktSUdrZXB
lZjVQeEE)

By clearing the label field or by entering nothing the script operates on 
anything inside your draft folder.

This changes something important. Before any emails inside the label would 
expect to be parse-able. If they could not then it was an error (because the 
user applied the label, they meant it to be sent at some point).

Now that we are operating on any emails inside the draft folder, only if we 
find the delimiter in the subject do we assume that the user was intending the 
email to be sent (and apply error label). Otherwise we silently ignore that 
mail.
```

Original comment by `blairk...@gmail.com` on 3 Dec 2011 at 6:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `blairk...@gmail.com` on 18 Dec 2011 at 9:30
- Changed state: **Fixed**
